doc/adrs/124_support_deck_representation.adoc

```diff
@@ -0,0 +1,168 @@
+= [ADR-124] Add the support for Deck representation
+
+== Context
+
+Deck is a new representation type to be added in Sirius web.
+It will allow to provide three kind of representation
+
+* Kanban
+* Daily
+* OKR
+
+== Decision
+
+For the front-end, the deck representation will rely on the https://github.com/rcdexta/react-trello/[react-trello project].
+
+We will create dedicated concepts for describing a Deck representation. It is called `Deck Description`. `Deck Description` concepts will be part of the `View` model.
+
+We will provide three instances/example of `Deck Description` corresponding to Kanban, Daily et OKR. These representation description will represent the `Task` meta-model.
+The `Task` meta-model is a new meta-model that will also be provided.
+
+
+== Specification
+
+=== Overview of a Deck representation
+
+A Deck representation displays vertical lanes in which there are cards.
+
+image::./images/124/deck_overview.png[]
+
+=== General actions related to representation
+
+Like others representations in Sirius web, the Deck representation can be created from contextual menu on semantic object displayed in the Explorer tree.
```

Comment:
Like other (no S)

Reply:
can be created from **a** contextual menu

Reply:
semantic object**s** displayed

Reply:
done